packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts

```diff
@@ -341,7 +337,19 @@ export const getStyles = (props: IDetailsRowStyleProps): IDetailsRowStyles => {
       className
     ],
 
+    isPadded: [
```

Comment:
Re: commit message, we should NEVER be reliant on ordering in css-in-js to determine what applies. The styles need to be done in a way where there is no overwriting at any one time. Example:

.listItem { /* no default color */ }
.listItemUnselected { color: gray; }
.listItemSelected { color: darkgray; }
.listItemSelectedHovered { color: black; }

The last three classes are all mutually exclusive, they should never be on the same element at the same time. The first one can be paired with any of the last three, though.